news/2020-04-24-spring-indoors-2020-fanart-contest-results.md

```diff
@@ -0,0 +1,135 @@
+
+[![](https://assets.ppy.sh/contests/92/banners/Sjao.jpg)](https://assets.ppy.sh/contests/92/winners/Sjao.png)
+
+Inked in some of the finest colouring work we've seen in a fanart contest so far, **2693** voters cast their eye towards [**Sjao**](https://osu.ppy.sh/users/7295733)'s Mocha attired in an airy dress as she admires her yard from her window.
+
+The vibrancy and tone work in this piece is just absolutely stunning, as is the attention to shadows and the particulars of light all around. Fantastic stuff!
+
+[![](https://assets.ppy.sh/contests/92/banners/Hecatia.jpg)](https://assets.ppy.sh/contests/92/winners/Hecatia.png)
+
+**2470** voters joined [**Hecatia**](https://osu.ppy.sh/users/8244635)'s pippi for some socially-distanced boba set to the gentle tones of sunset.
+
+We unofficially titled this piece as 'the Last Straw'. You will let her have the last straw, right?
+
+[![](https://assets.ppy.sh/contests/92/banners/157.jpg)](https://assets.ppy.sh/contests/92/winners/157.png)
+
+**2403** voters understood that M is for Mocha (and for midriff) in [**157**](https://osu.ppy.sh/users/9315038)'s take on our most adorable mascot lazing around in a storm of sunlit petals.
+
+The hoodie she is wearing is *clearly* an official osu! hoodie. No word yet on when official osu! thigh highs will join the merch ensemble.
+
+[![](https://assets.ppy.sh/contests/92/banners/Aoki%20chiaki.jpg)](https://assets.ppy.sh/contests/92/winners/Aoki%20chiaki.png)
+
+**2160** people snuck a sneak peek into Chirou's room as imagined by [**Aoki chiaki**](https://osu.ppy.sh/users/7692958), and were gobsmacked by its cleanliness and botanical theme.
+
+That, and the tiny mask-wearing catgirl chibi pippi that has claimed her computer chair.
+
+[![](https://assets.ppy.sh/contests/92/banners/Phillette.jpg)](https://assets.ppy.sh/contests/92/winners/Phillette.png)
+
+**1940** voters watched along with Mocha as she gazes upon pippi's feeble attempts to conquer the ancestor of her most beloved game mode, courtesy of [**Phillette**](https://osu.ppy.sh/users/9457944)'s artistic skills.
+
+We presume Mocha is also snacking on what looks to be green tea pocky in the process. The old saying "you are what you eat" takes on a different meaning when you realize that both said pocky and Mocha's hair share the same colour. Hmm.
+
+[![](https://assets.ppy.sh/contests/92/banners/HEKUSODASU.jpg)](https://assets.ppy.sh/contests/92/winners/HEKUSODASU.png)
+
+[**HEKUSODASU**](https://osu.ppy.sh/users/9044692) drew **1901** tiny chibi pippis doing tiny chibi pippi things. Oh, and that many people voted for the entry as well.
+
+[![](https://assets.ppy.sh/contests/92/banners/Eternity%20Youmu.jpg)](https://assets.ppy.sh/contests/92/winners/Eternity%20Youmu.png)
+
+In perhaps the most realistic take of all the entries this far, [**Eternity Youmu**](https://osu.ppy.sh/users/1012268)'s take on a napping (she's not sick because she stayed indoors, okay?!) pippi won the hearts of **1765** voters.
```

Comment:
Her thermometer says 38C which is 100.4F which is the exact number the CDC defines as having a fever. pippi's sick. this comment isn't accurate. I hope she gets well soon.

Reply:
:ohno: